### DISCUSSION TITLE
lsp-server for fastn

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#609] [Ideas & RFCs]
https://github.com/ebkalderon/tower-lsp

**Comment by gsalunke** [comment]
Ref.: https://microsoft.github.io/language-server-protocol/

**Reply by amitu** [reply]
https://github.com/IWANABETHATGUY/tower-lsp-boilerplate

**Reply by amitu** [reply]
https://www.alanwsmith.com/pages/rust-lsp-bookmarks--2w2zrczh/

**Reply by amitu** [reply]
https://www.osohq.com/post/building-vs-code-extension-with-rust-wasm-typescript
